### DISCUSSION TITLE
Mkdocs online documentation will not update but local documentation will

### DISCUSSION THREAD

**Discussion by galenseilis** [mkdocs/mkdocs#3447] [Q&A] [answered]
Please see https://github.com/galenseilis/Mjolnir/issues/1 for a detailed description.

**Comment by oprypin** [comment, marked as answer]
Hi.
You have a mkdocs source site on the "main" branch. You've been running `mkdocs gh-deploy` to deploy it but that was failing. Maybe running `git fetch` would resolve that.
I see that you also have a CI job that pushes to gh-pages, and it seems to work.
I have no idea why the website wasn't refreshing. But it is refreshed now as far as I see. Maybe try Ctrl+F5 in the browser to clear cache.

Anyway for any issue that you have, you can completely eliminate mkdocs from the equation. If the local build works but remote deploy doesn't, just copy the built files to the deployment destination in any way that you like, and see if that succeeds.

You were also mentioning that you might have to move to Sphinx and readthedocs. Well why not MkDocs and readthedocs then

**Reply by galenseilis** [reply]
Hi @oprypin ,

Thank you for looking into my issue. I really appreciate it!

Your hunch to clear the browser cache was spot on. After clearing the cache associated with `galenseilis.github.io` I found that the site refreshed appropriately. I'll be sure to keep my eyes out for this issue going forward.

At this point there is no reason I shouldn't continue with MkDocs. It makes beautiful documentation with relatively little effort, which is what attracted me to it in the first place.

Thanks!
